react-components/src/apps/listing/AppGridListing.js

```diff
@@ -250,7 +249,7 @@ function getTableColumns(deletable, enableMenu) {
         },
         {
             name: AppFields.SYSTEM.fieldName,
-            enableSorting: true,
+            enableSorting: false,
```

Comment:
disabled sorting by `system ID` field since there is a bug in app listing service when sorting using this field.